examples/pagerank/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
I'm not sure if we need a CMakeList here. Seems put these to CMakeLists in root path and make example as a new target would be better. 
A new CMakeList.txt here would bring some cons:
- it need user/develop to **install** GraphAr first, not just make
- it's building need to build arrow again.

Reply:
Yes. I think what you suggested is the in-project examples. What I want to show here is a new application employing the GAR library, in which case, users should first install GAR, and then build their own application projects.